src/ResourceManager/Sql/documentation/current-breaking-changes.md

```diff
@@ -42,6 +42,9 @@
 
 The following cmdlets were affected this release:
 
+**Get-AzureRmSqlServiceObjective**
```

Comment:
Take this out for now, it should be added back in when the parameter is removed in May.

Reply:
done.